src/server/plugins/views.ts

```diff
@@ -45,7 +62,12 @@ function nunjucksContext(
     location: app?.location,
     phaseTag: config.phaseTag,
     previewMode: isPreviewMode ? params?.state : undefined,
-    slug: params?.slug
+    slug: params?.slug,
+
+    getAssetPath(asset: string) {
```

Comment:
worth keeping this consistent with forms-designer?

https://github.com/DEFRA/forms-designer/blob/82a85574084528c494957415ae32d6b960f43855/designer/server/src/common/nunjucks/context/index.js#L39

Reply:
Mind if we do it the other way? I'll update **forms-designer** to match this

I've opened a PR with all our changes in with CDP if you want to see a unified version:

* https://github.com/DEFRA/cdp-node-frontend-template/pull/82

Reply:
Yep that's fine.